plugin/bbb/lib/bbb_plugin.class.php

```diff
@@ -45,7 +45,7 @@ class BBBPlugin extends Plugin
     protected function __construct()
     {
         parent::__construct(
-            '2.9',
+            '2.10',
             'Julio Montoya, Yannick Warnier, Angel Fernando Quiroz Campos, Jose Angel Ruiz',
```

Comment:
Feel free to add the name of the main contributor at the end of the list, if you want. This will mostly be used by people checking if they know one of these people, to ask them for help.